### DISCUSSION TITLE
maximal pattern support

### DISCUSSION THREAD

**Discussion by MotifDscvry** [rasbt/mlxtend#1015] [General]
I find the fmax of mlxtend.frequent_patterns does not calculate the pattern support right. Any suggestion?

**Comment by rasbt** [comment]
could you share an example of the expected and the actual outputs? That would help understand what's going on

**Comment by shenxiangzhuang** [comment]
Hi @MotifDscvry , what version of mlxtend are you using? It does has a bug in version before **0.17.2** as addressed in #709. 
If you are using an old version, try to upgrade to the newest version and try it again.
